app/boards/shields/qaz/Kconfig.shield

```
@@ -1,5 +1,5 @@
-# Copyright (c) 2020 TJ Campie
+# Copyright (c) 2020 The ZMK Contributors
 # SPDX-License-Identifier: MIT
- 
- config SHIELD_MY_BOARD
-    def_bool $(shields_list_contains,qaz)
+
+config SHIELD_MY_BOARD
```

I missed this in my first review of this, this should be:

```suggestion
config SHIELD_QAZ
```